AllASPFuncs.py

```diff
@@ -1116,8 +1102,9 @@ def get_SN_SED(SNID, date, sn_path):
     mjd = sed_table['mjd']
     bestindex = np.argmin(np.abs(np.array(mjd) - date))
     if np.min(np.abs(np.array(mjd) - date)) > 10:
-        print('WARNING: No SED data within 10 days of date. \n \
-            The closest SED is ' + str(np.min(np.abs(np.array(mjd) - date))) + ' days away.')
+        Lager.warn('WARNING: No SED data within 10 days of date. \n \
+            The closest SED is ' + str(np.min(np.abs(np.array(mjd) - date))) +
+                                       ' days away.')
```

Comment:
As a general practice for having tests and error messages, use a variable to make sure you have the same quantity and the error message and the drift don't fall out of sync.

You use the quantities
`10` and
`np.min(np.abs(np.array(mjd) - date))`
in both the test and the error message.  I suggest storing each as a variable, and then use the same variable in the test and the error message.